### PR TITLE
Comment flashloan handling

### PR DESCRIPTION
**Pull request by MerlinEgalite**
Fixes https://github.com/cantinasec/review-morpho-blue-1/issues/107

### CODE REVIEW

**Comment by julien-devatom** [comment]
Wrong linked issue no?

**Comment by MerlinEgalite** [comment]
> Wrong linked issue no?

I can't fix it lol

File: src/MetaMorpho.sol [hunk 1]
```diff
@@ -792,6 +792,11 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         (uint256 totalSupplyAssets, uint256 totalSupplyShares, uint256 totalBorrowAssets,) =
             MORPHO.expectedMarketBalances(marketParams);
 
+        // In case a call to the vault is done during a flashloan on Morpho Blue we can have:
+        // totalSupplyAssets - totalBorrowAssets > ERC20(marketParams.loanToken).balanceOf(address(MORPHO)
+        // To keep ERC4626 compliance we must return the available liquidity at the moment:
+        // ERC20(marketParams.loanToken).balanceOf(address(MORPHO)
+        // Hence, the min in the following statement:
```
Comment (Rubilmax) [inline src/MetaMorpho.sol:799]:
```suggestion
        // Inside a flashloan callback, liquidity on Morpho Blue can be limited to the singleton's balance.
```

Comment (MerlinEgalite) [reply src/MetaMorpho.sol:799]: The can sounds weird, it seems like this is a feature

**Review by QGarchery — APPROVED** [review]

**Review by Rubilmax — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Pull request merged**